scripts/push-docker-images.sh

```diff
@@ -32,12 +32,21 @@ if [[ "${MUTABLE_TAG:-}" == "" ]]; then
     fi
 fi
 
+echo "Building Athens image for registry ${REGISTRY}"
+echo "Version ${VERSION}"
+echo "Mutable tag ${MUTABLE_TAG}"
+
 REPO_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null && pwd )/"
 
+echo "docker build -t ${REGISTRY}proxy:${VERSION} -f ${REPO_DIR}cmd/proxy/Dockerfile ${REPO_DIR}"
```

Comment:
How about doing a `set -x` at the top of the script instead? I'm concerned that we are going to get drift between the logged lines and the actual commands run which would be extra hard to debug.

Reply:
I think we're already do that.
https://github.com/gomods/athens/blob/657c9d04a35197a23667a064804deda38bcc0256/scripts/push-docker-images.sh#L4